README.md

```diff
@@ -1,8 +1,15 @@
 Codesis Website project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Getting Started
+1. Goto the [codesis-website](https://github.com/The-Codesis/codesis-website/) repository
+2. Click on the Fork button in the upper right corner.
+3. Clone the forked repository on your local machine
+```
+git clone https://github.com/<your username>/codesis-website.git
+```
+4. To install the dependencies ,in the project directory you can run:
+### `yarn install`
 
-## Available Scripts
-
-In the project directory, you can run:
+5. Once the dependencies are installed,run:
 
 ### `yarn start`
 
```

Comment:
This is really good. I can merge it at this stage. But it will be really great if you can add that how someone can submit a pull request by making another branch here (Just a suggestion).

Reply:
Yes will do it in a while!

Reply:
@NidhiKJha I think we can tackle that in a separate PR ?

Reply:
@rahul799 we can, but it's just a suggestion. It will make his pr complete and we can close this issue.